app/assets/stylesheets/pageflow/themes/default/loading_spinner.scss

```diff
@@ -59,8 +72,11 @@ $loading-spinner-image-file-extension: "png" !default;
     // scss-lint:enable ImportantRule
 
     &:before {
-      background-position: 50% 50%;
+      background-position: $loading-spinner-background-position;
       background-repeat: no-repeat;
+      @if $loading-spinner-background-size {
+        background-size: $loading-spinner-background-size;
+      }
```

Comment:
This `@if` is actually not needed since properties with `null` values are omitted automatically.